WebAPI/Controllers/PermissionsController.cs

```diff
@@ -0,0 +1,81 @@
+using Infrastructure;
+using Infrastructure.DTO.Permissions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using System.Data;
+
+namespace WebAPI.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")]
+    public class PermissionsController : ControllerBase
+    {
+
+        private LANHQDbContext _context;
+
+        public PermissionsController(LANHQDbContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Permissions/Roles
+        [HttpGet("Roles")]
+        public IActionResult GetRoles()
```

Comment:
Pour tout tes endpoints tu vas probablement vouloir ajouter plus d'attribue pour rendre ton swagger plus détailler.

Exemple : https://stackoverflow.com/questions/52883466/how-to-add-method-description-in-swagger-ui-in-webapi-application